packages/algolia/src/client.ts

```diff
@@ -66,13 +86,20 @@ export class AlgoliaSearchClient {
     return this.index.search<EntityRecord<T>>(query, options);
   }
 
+  public static toPayload(
```

Comment:
I'm not sure I understand this function, is it just for tests? if so - this is a static function, surely you could just make it a helper function inside the test directory?